src/constants.hpp

```diff
@@ -1,9 +1,9 @@
 #ifndef CONSTANTS_HPP
 #define CONSTANTS_HPP
 
-inline constexpr int SCREEN_WIDTH = 800;
-inline constexpr int SCREEN_HEIGHT = 600;
+inline constexpr int SCREEN_WIDTH = 769;
+inline constexpr int SCREEN_HEIGHT = 673;
 
-const int GEM_SIZE = 32;
+const int GEM_SIZE = 96;
```

Comment:
You might need to use `inline constexpr` instead? I don't remember to be honest, does it compile?